build.xml

```diff
@@ -227,6 +227,131 @@ sudo /etc/init.d/virtuoso-opensource-6.1 restart"
               passthru="true"/>
     </target>
 
+    <target name="download-apache-solr">
+        <if>
+            <available file="${solr.dir}" type="dir" property="solr.dir.available" />
+            <then>
+                <echo message="Apache Solr installation directory exists (${solr.dir}), skipping download." />
+            </then>
+            <else>
+                <if>
+                    <available file="${tmp.dir}/${solr.tarball.filename}" type="file" property="solr.tarball.available" />
+                    <then>
+                        <echo message="Apache Solr tarball already downloaded to ${tmp.dir}/${solr.tarball.filename}, skipping download." />
+                    </then>
+                    <else>
+                        <mkdir dir="${tmp.dir}" />
+                        <httpget url="${solr.download.url}" dir="${tmp.dir}" />
+                    </else>
+                </if>
+                <!-- Use the faster native untar on UNIX systems. -->
+                <if>
+                    <os family="unix" />
+                    <then>
+                        <mkdir dir="${solr.vendor.dir}" />
+                        <exec
+                            command='tar xzf "${tmp.dir}/${solr.tarball.filename}" -C "${solr.vendor.dir}"'
+                            dir="${project.basedir}"
+                            passthru="true"
+                            checkreturn="true"
+                        />
+                    </then>
+                    <else>
+                        <untar file="${tmp.dir}/${solr.tarball.filename}" todir="${solr.vendor.dir}" />
+                    </else>
+                </if>
+            </else>
+        </if>
+        <!-- Create a symlink of the solr exec file to vendor/bin directory. -->
+        <if>
+            <available file="${project.basedir}/vendor/bin/solr" type="file" property="file.Exists"/>
+            <then>
+                <echo message="Solr binary already exists in ${project.basedir}/vendor/bin directory. Skip creating symlink" />
+            </then>
+            <else>
+                <symlink
+                        target="${solr.bin}"
+                        link="${project.basedir}/vendor/bin/solr"
```

Comment:
This will need to be updated too then, you can set the target to `${solr.dir}/bin/solr`, that is consistent with the other targets like `start-apache-solr` etc.